packages/git-wrapper/tests/AbstractGitWrapperTestCase.php

```diff
@@ -49,6 +49,7 @@ protected function setUp(): void
     {
         $this->smartFileSystem = new SmartFileSystem();
         $this->gitWrapper = new GitWrapper('git');
+        $this->gitWrapper->setEnvVar('LC_ALL', 'C');
```

Comment:
What is this for?

Reply:
I have Polish locale on my computer, so some tests failed because they analyze git command output assuming they're in english. This eliminates this problem. But yeah, it's not connected with the "provide" issue. Do you want me to remove this commit from this PR?

Reply:
I see. In that case you can keep it. But please add some comment above, to explain why